basicSwift.playground/Contents.swift

```diff
@@ -1,3 +1,194 @@
 import UIKit
 
-var str = "Hello, playground"
+func sayHi() {
+    print("Hi")
+}
+// v1
+var arrayJournal = [String]()
+
+func addStudent (name: String, profession: String) {
+    arrayJournal.append(name)
+    arrayJournal.append(profession)
+    print(arrayJournal)
+}
+
+addStudent(name: "John", profession: "Ios dev")
+addStudent(name: "Jack", profession: "Andriod Dev")
+addStudent(name: "Ivan", profession: "Java")
+arrayJournal
+
+//v2
+func calculator (number1: Int, number2: Int) -> Int {
```

Comment:
```suggestion
func calculator(number1: Int, number2: Int) -> Int {
```
я говорил тут не ставить пробед...

Reply:
sluchayno, sorry